activerecord/test/cases/scoping/default_scoping_test.rb

```diff
@@ -104,6 +104,30 @@ def test_unscope_overrides_default_scope
     assert_equal expected, received
   end
 
+  def test_unscope_overrides_a_stringified_scope
+    expected = Developer.all.collect { |dev| [dev.name, dev.id] }
+    received = Developer.named_david_or_jamis_stringified.unscope(where: "name = 'David' OR name = 'Jamis'").collect { |dev| [dev.name, dev.id] }
```

Comment:
How does this work with interpolated array values? `["name = ? OR name = ?", "David", "Jamis"]`

Reply:
By looking at this [line](https://github.com/rails/rails/blob/master/activerecord/lib/active_record/relation/query_methods.rb#L487) the implementation of `unscope` does not handle interpolated arrays atm. 
But I totally agree with you that interpolation makes much more sense.
I would like to know your opinion that should we add the interpolation support in the unscope method because I didn't find any stringified queries in this whole file.